src/prototype/main.py

```diff
@@ -0,0 +1,85 @@
+import sys
+import re
+import os
+from PyQt5.QtWidgets import QApplication, QMainWindow
```

Comment:
A new line between Python's default packages, packages from the Internet and our packages would be nice.